staging/src/k8s.io/client-go/transport/cache.go

```diff
@@ -80,6 +81,7 @@ func (c *tlsTransportCache) get(config *Config) (http.RoundTripper, error) {
 		// Ensure we only create a single transport for the given TLS options
 		c.mu.Lock()
 		defer c.mu.Unlock()
+		defer metrics.TransportCacheEntries.Observe(context.Background(), float64(len(c.transports)))
```

Comment:
Er what exactly is the purpose of using a context if it is always set to background?

Reply:
copy paste thing